### PR TITLE
feat:add the code create way is not under GOPATH

### PR DESCRIPTION
**Pull request by Nihilism0**
#### What type of PR is this?
feat

#### What this PR does / why we need it (en: English/zh: Chinese):

en: add the code create way is not under GOPATH(thrift)
zh:提供非GOPATH下的项目生成代码的步骤

### CODE REVIEW

File: content/zh/docs/hertz/tutorials/toolkit/usage/usage-thrift.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ go mod edit -replace github.com/apache/thrift=github.com/apache/thrift@v0.13.0
 go mod tidy
 ```
 
+```bash
```
Comment (FGYFFFF) [inline content/zh/docs/hertz/tutorials/toolkit/usage/usage-thrift.md:45]: protobuf 和英文版本都加一个类似的吧~~~

**Comment by GuangmingLuo** [comment]
参考其它 PR 优化下 PR title

**Review by FGYFFFF — APPROVED** [review]
lgtm

File: content/en/docs/hertz/tutorials/toolkit/usage/usage-thrift.md [hunk 2]
```diff
@@ -42,6 +42,20 @@ go mod edit -replace github.com/apache/thrift=github.com/apache/thrift@v0.13.0
 go mod tidy
 ```
 
+```bash
+// Execute is not under GOPATH
+//option 1,you do not have a go.mod, add go mod name after - module
```
Comment (welkeyever) [inline content/en/docs/hertz/tutorials/toolkit/usage/usage-thrift.md:47]: ‘- module’ -> `-module`?

Try to use code block.

Comment (welkeyever) [reply content/en/docs/hertz/tutorials/toolkit/usage/usage-thrift.md]: Get, then just remove the extra blank between - and module

Comment (Nihilism0) [reply content/en/docs/hertz/tutorials/toolkit/usage/usage-thrift.md]: I have removed it. You can see Files changed

**Review by welkeyever — APPROVED** [review]
LGTM, thx~

**Pull request merged**